.env.local

```
@@ -83,7 +83,7 @@ FLEXPRICE_EVENT_PROCESSING_LAZY_CONSUMER_GROUP="local_events_lazy_consumer"
 # Add more keys as needed: {"<sha256_of_key>": {"tenant_id": "...", ...}}
 FLEXPRICE_AUTH_PROVIDER="api_key"
 FLEXPRICE_AUTH_API_KEY_HEADER="x-api-key"
-FLEXPRICE_AUTH_API_KEY_KEYS="{\"0cfd568f22158887f8b77bc019fb245b1f14077b80936342b052985efa7de46c\":{\"tenant_id\":\"00000000-0000-0000-0000-000000000000\",\"user_id\":\"local-dev-user\",\"name\":\"local-dev-key\",\"is_active\":true}}"
+FLEXPRICE_AUTH_API_KEY_KEYS="{"0cfd568f22158887f8b77bc019fb245b1f14077b80936342b052985efa7de46c":{"tenant_id":"00000000-0000-0000-0000-000000000000","user_id":"local-dev-user","name":"local-dev-key","is_active":true}}"
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)